src/openforms/js/components/admin/form_design/variables/VariablesEditor.stories.js

```diff
@@ -507,7 +507,7 @@ export const ConfigurePrefill = {
     await userEvent.click(userDefinedVarsTab);
 
     // open modal for configuration
-    const editIcon = canvas.getByTitle('Edit prefill configuration');
+    const editIcon = canvas.getByTitle('Prefill instellen');
```

Comment:
I thought it was customary to use English and then add the Dutch translation (at least for Python code)? Is this different for JS?

Reply:
same principle applies, however dealing with non-default languages in storybook is a massive pain in the rear, so for these interaction tests we just use the NL strings.